content/concepts/consensus-network/consensus-network.md

```diff
@@ -2,30 +2,33 @@
 <!--{# TODO: Are we learning about the Consensus Network or the Consensus Process? Both? Is one a child of the other -- or is the parent concept simply Consensus? Also, need to be careful to differentiate this content from the content we provide in Concepts > XRP Ledger Basics > Consensus. What should I expect to learn here - and how is it different from what I will learn there? #}-->
 The primary role of consensus is for participants in the process to agree on which XRP Ledger transactions are to be processed as a group to resolve the double spend problem. This section covers concepts that help you learn the "what" and "why" behind fundamental aspects of consensus in the XRP Ledger.
 
-* **[Consensus Principles and Rules](x)**
+* **[Consensus Principles and Rules](consensus-principles-and-rules.html)**
 
       Understand the principles and rules involved in reaching consensus in the XRP Ledger, including some background on the double-spend problem and details about how consensus works.
 <!--{# TODO: this language needs to be refined based on what we're actually focusing on here. Are these the principles and rules of consensus, the network, or the process? https://developers.ripple.com/concept-reaching-consensus.html #}-->
 
+<!--{#
 * **[Validators and the Trust Topology](x)**
-<!--{# TODO: need this intro text - what is this about? What page does it link to? New content? #}-->
+ TODO: need this intro text - what is this about? What page does it link to? New content? #}-->
```

Comment:
If we move the "TODOs" from here into the config file, we can delete this file entirely and switch to using a fully auto-generated landing page